connman/unit/test-globalproxy.c

```diff
@@ -289,15 +289,15 @@ DAPolicy* da_policy_new_full(const char* spec, const DA_ACTION* actions)
 // Needed to control permissions independent of reality
 void da_policy_unref(DAPolicy* policy)
 {
-	g_assert_cmphex((uint)policy, ==, POLICY_FAKE_POINTER);
```

Comment:
nit: While 'size_t' might silence warnings in this kinds of situations in some environments... there are 'intptr_t' and 'uintptr_t' c types that are meant for cases like this. Or, maybe even better, since this is glib code: GPOINTER_TO_UINT() macro could be used?

Reply:
Actually `GPOINTER_TO_UINT()`  is ideal here. Changed and force-pushed.